spec/system/kudo_received_count_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Received kudo count', type: :system do
```

Comment:
I'd add the check to the employee's giving kudo spec as this case (just checking if the points counter is being displayed)